vendor/assets/stylesheets/dvl/core/grid.scss

```diff
@@ -1,95 +1,51 @@
-/* ================================================================== *\
-   Grids ($grids)
-\* ================================================================== */
+// Grid system
 
-$columnWidth: 8.333%;
-$gutterWidth: 2rem;
+$columnWidth: 6.25%;
+$gutterWidth: 2.0833333%;
```

Comment:
Can you use fractions so these numbers don't look like "magic numbers"? At the very least, add a comment or two